layers/enums.go

```diff
@@ -103,6 +103,7 @@ const (
 	LinkTypeSLIP           LinkType = 8
 	LinkTypePPP            LinkType = 9
 	LinkTypeFDDI           LinkType = 10
+	LinkTypeOVPN           LinkType = 12
```

Comment:
This will be covered in google/gopacket (confer https://github.com/google/gopacket/pull/567)